include/mbedtls/mbedtls_config.h

```diff
@@ -1045,6 +1045,16 @@
  */
 #define MBEDTLS_PK_PARSE_EC_EXTENDED
 
+/**
+ * \def MBEDTLS_PK_PARSE_EC_COMPRESSED
+ *
+ * Enable the support for parsing public keys of type Short Weierstrass
+ * (MBEDTLS_ECP_DP_SECP_XXX and MBEDTLS_ECP_DP_BP_XXX) which are using the
+ * compressed point format.
+ * Please see MBEDTLS_ECP_PF_COMPRESSED in ecp.h for limitations details.
```

Comment:
The relation between this define and `MBEDTLS_ECP_PF_COMPRESSED` is not explained.

Reply:
Oh, I see, so it's not a problem about the description itself as I thought initially, but it's more related to the automatic documentation generation. Am I correct?

Reply:
Ok understood. Does the [last commit](https://github.com/Mbed-TLS/mbedtls/pull/7717/commits/3cd4ef7a7abcd1d7f5645e69da0ebf7782954e62) cover this gap or should it be reworked further?

Reply:
Yes, it's OK, now you're providing the full info and the source of it, regardless of the relation between the two places.